.gitignore

```diff
@@ -1,7 +1,12 @@
 /node_modules
 .idea
-**/*.private.env.json
-**/deploy_account.json
-**/*.pem
```

Comment:
Isn't the `**/*` syntax for catching files in any subdirectory? I worry that, without it, it will only catch files in the base directory. I'd have to read the gitignore docs; I don't use it frequently enough to remember.

Reply:
> Isn't the **/* syntax for catching files in any subdirectory?

Sorta.

> If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.

via: https://git-scm.com/docs/gitignore#_pattern_format

So these are all equivalent:
- `*.pem`
- `**/*.pem`
- `**/**/*.pem`
- `**/**/**/*.pem`
- etc.

I went for the first option.